cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/SteadyStateHypothesisExport.java

```diff
@@ -0,0 +1,629 @@
+    private static final String GENERATING_UNIT_PROPERTY = Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + "GeneratingUnit";
+
+    private SteadyStateHypothesisExport() {
+    }
+
+    public static void write(Network network, XMLStreamWriter writer, CgmesExportContext context) {
+        final Map<String, List<RegulatingControlView>> regulatingControlViews = new HashMap<>();
+        String cimNamespace = context.getCimNamespace();
+
+        try {
+            CgmesExportUtil.writeRdfRoot(context.getCimVersion(), writer);
+
+            if (context.getCimVersion() == 16) {
+                CgmesExportUtil.writeModelDescription(writer, context.getSshModelDescription(), context);
+            }
+
+            writeEnergyConsumers(network, cimNamespace, writer);
+            writeEquivalentInjections(network, cimNamespace, writer);
+            writeTapChangers(network, cimNamespace, regulatingControlViews, writer);
+            writeSynchronousMachines(network, cimNamespace, regulatingControlViews, writer);
+            writeShuntCompensators(network, cimNamespace, regulatingControlViews, writer);
+            writeStaticVarCompensators(network, cimNamespace, regulatingControlViews, writer);
+            writeRegulatingControls(regulatingControlViews, cimNamespace, writer);
+            writeGeneratingUnitsParticitationFactors(network, cimNamespace, writer);
+            // FIXME open status of retained switches in bus-branch models
+            writeSwitches(network, cimNamespace, writer);
+            // TODO writeControlAreas
+            writeTerminals(network, cimNamespace, writer);
+
+            writer.writeEndDocument();
+        } catch (XMLStreamException e) {
+            throw new UncheckedXmlStreamException(e);
+        }
+    }
+
+    private static void writeSwitches(Network network, String cimNamespace, XMLStreamWriter writer) {
+        for (Switch sw : network.getSwitches()) {
+            writeSwitch(sw, cimNamespace, writer);
+        }
+    }
+
+    private static final String ALIAS_TYPE_TERMINAL_1 = Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.TERMINAL + "1";
+    private static final String ALIAS_TYPE_TERMINAL_2 = Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.TERMINAL + "2";
+
+    private static void writeTerminals(Network network, String cimNamespace, XMLStreamWriter writer) {
+        for (Connectable<?> c : network.getConnectables()) {
+            for (Terminal t : c.getTerminals()) {
+                writeTerminal(t, c, cimNamespace, writer);
+            }
+        }
+        for (Switch sw : network.getSwitches()) {
+            // Terminals for switches are exported as always connected
+            // The status of the switch is "open" if any of the original terminals were not connected
+            // An original "closed" switch with any terminal disconnected
+            // will be exported as "open" with terminals connected
+            sw.getAliasFromType(ALIAS_TYPE_TERMINAL_1)
```

Comment:
I am not sure to get how we should model an open switch. Do we have to modify only the status and keep the terminals connected?